autoscheduler/scraper/models/department.py

```diff
@@ -1,9 +1,17 @@
 from django.db import models
 
+def generate_department_id(code: str, term: str):
+    """ Generates the department id as {code}-{term} """
+
+    return "".join((code, '-', term))

```

Comment:
Super simple function, thought join was the most more readable than doing `{code}-{term]`, could also change to `code + '-' + term`, bit of a nitpick though